web/directory/static/css/dev/style.css

```diff
@@ -0,0 +1,129 @@
+/* line 11, ../../sass/style.scss */
```

Comment:
Not a blocker, but web/directory/static has a lot of duplication with web/directory/static-compiled that I'd like to see removed from the repo.  If they both need to exist for some reason, then there should be one version checked in and the other should be copied as part of the `make build` process.

Reply:
See comment above.